tools/bin/ci_credentials.sh

```diff
@@ -46,3 +46,4 @@ write_standard_creds source-slack-singer "$SLACK_TEST_CREDS"
 write_standard_creds source-stripe-singer "$STRIPE_INTEGRATION_TEST_CREDS"
 write_standard_creds source-twilio-singer "$TWILIO_TEST_CREDS"
 write_standard_creds source-zendesk-singer "$ZENDESK_SECRETS_CREDS"
+write_standard_creds source-intercom-singer "$INTERCOM_SECRETS_CREDS"
```

Comment:
could you also add a line to `.github/workflows/test-command.yaml` like here https://github.com/airbytehq/airbyte/blob/master/.github/workflows/test-command.yml#L70